gologger.go

```diff
@@ -22,6 +22,7 @@ var (
 	}
 	// DefaultLogger is the default logging instance
 	DefaultLogger *Logger
+	addTimestamp  bool
```

Comment:
shouldn't this be defined as field of `Logger`?

Reply:
What about using `DefaultLogger` to check `addTimestamp`?